backend/seed_data/quiz_questions.yaml

```
@@ -466,3 +466,476 @@ quiz_questions:
+      - option_text: "When you need to evaluate postfix expressions"
+        is_correct: false
+        option_type: CONCEPT
+      - option_text: "When you need to create a copy of the tree"
+        is_correct: true
+        option_type: CONCEPT
+      - option_text: "When you need to retrieve elements in sorted order"
+        is_correct: false
+        option_type: CONCEPT
+      - option_text: "When you need to traverse nodes level by level"
+        is_correct: false
+        option_type: CONCEPT
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)